05-setting-up.Rmd

```diff
@@ -9,35 +9,169 @@ ottrpal::set_knitr_image_path()
+
+The most important aspects of your project organization scheme is that it:  
+
+- Is [project-oriented](https://www.tidyverse.org/blog/2017/12/workflow-vs-script/) [@Bryan2017].  
+- Follows consistent patterns [@Shapiro2021].  
+- Is easy for you and others to find the files you need quickly [@Shapiro2021].  
+- Minimizes the likelihood for errors (like writing over files accidentally) [@Shapiro2021].  
+- Is something maintainable [@Shapiro2021]!
+
 ```{r, fig.align='center', out.width="100%", echo = FALSE, fig.alt= "Major point!! example image"}
-ottrpal::include_slide("https://docs.google.com/presentation/d/1MNHf8JpolaEP_vQ_kB-1xRBF9wo3haCArRu117hBoHA/edit#slide=id.g2fea8805c08_0_426")
+ottrpal::include_slide("https://docs.google.com/presentation/d/1MNHf8JpolaEP_vQ_kB-1xRBF9wo3haCArRu117hBoHA/edit#slide=id.g2fea8805c08_0_421")
+```
+
+### READMEs!
+
+READMEs are also a great way to help your collaborators get quickly acquainted with the project.
+
+```{r, fig.align='center', echo = FALSE, fig.alt= "Avi is looking at a set of project files that include a file called a ‘README.md’. Avi says 'I had no idea where to start with this analysis that Ruby sent me to review, but then I saw she included a README and that saved me so much time and effort in getting started!'"}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1LMurysUhCjZb7DVF6KS9QmJ5NBjwWVjRn40MS9f2noE/edit#slide=id.gf8379bb805_0_11")
 ```
 
+READMEs stick out in a project and are generally universal signal for new people to the project to start by READing them. GitHub automatically will preview your file called "README.md" when someone comes to the main page of your repository. This further encourages people looking at your project to read the information in your README.
+
+**Information that should be included in a README:**
+
+1) General purpose of the project
+2) Instructions on how to re-run the project
+3) Lists of any software required by the project
+4) Input and output file descriptions
+5) Descriptions of any additional tools included in the project
+
+You can take a look at this [template README](https://raw.githubusercontent.com/jhudsl/Reproducibility_in_Cancer_Informatics/main/resources/README-template.md) to get your started.
+
+#### More about writing READMEs:
+
+- [How to write a good README file by Hillary Nyakundi](https://www.freecodecamp.org/news/how-to-write-a-good-readme-file/)
+- [How to write an awesome README by Navendu Pottekkat](https://towardsdatascience.com/how-to-write-an-awesome-readme-68bf4be91f8b)
+
+#### Examples of good READMEs:
+
+- https://github.com/stephaniehicks/qsmooth
+- https://github.com/lcolladotor/derfinder
+- https://github.com/tidyverse/dplyr
+
```

Comment:
```suggestion

#### Licensing

Adding information about a license is not always required, but it can be a good idea. If you put your code on GitHub, then the default copyright laws apply.  According to GitHub:

> "You retain all rights to your source code and no one may reproduce, distribute, or create derivative works from your work. If you're creating an open source project, we strongly encourage you to include an open source license."

::: dictionary
Open source software or code means that it is distributed with a license that allows others to reuse or adapt your code for other purposes. This is very helpful to advance science and technology. 
:::

Check out this great resource on [options for licenses](https://choosealicense.com/) to help you choose which license is right for your project.
```

Reply:
Good call!